feat(changelog): jobs-added-serverless-jobs-scheduling-cron-available 2024-02-21 (#2778)
scaleway/docs-content: changelog/february2024/2024-02-21-jobs-added-serverless-jobs-scheduling-cron-available.mdx

---
title: Serverless Jobs scheduling (CRON) available
status: added
author:
    fullname: 'Join the #serverless-jobs-beta channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-02-21
category: serverless
product: jobs
---

Serverless Jobs now supports scheduling!

> Optimize repetitive tasks and resource usage using CRON syntax (with custom timezone support). You can now automate your data processing, nightly data backups or periodic data analysis.

Reviewer: ```suggestion
Optimize repetitive tasks and resource usage using CRON syntax (with custom time zone support). You can now automate your data processing, nightly data backups or periodic data analysis.
```

---
title: Serverless Jobs scheduling (CRON) available
status: added
author:
    fullname: 'Join the #serverless-jobs-beta channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-02-21
category: serverless
product: jobs
---

Serverless Jobs now supports scheduling!

> 
> 

Reviewer 2: ```suggestion

```

> Using CRON syntax (with custom timezone support) you can now optimize repetitive tasks and resource usage. It allows more use cases such as automation of data processing, nightly data backups or weekly data analysis. 

Reviewer: ```suggestion
Optimize repetitive tasks and resource usage using CRON syntax (with custom timezone support). You can now automate your data processing, nightly data backups or periodic data analysis.
```
